dvc/command/gc.py

```diff
@@ -47,6 +56,7 @@ def run(self):
             force=self.args.force,
             jobs=self.args.jobs,
             repos=self.args.repos,
+            workspace=self.args.workspace,
```

Comment:
A general thought, not about this PR, - we should stop automatically making command line options into boolean kwargs, e.g. using `set()` would be more appropriate here.